examples/advanced/random_forest/job.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Recipe-based Federated Random Forest using XGBoost Tree-based Bagging.
+
+This example demonstrates federated Random Forest training using NVFlare's recipe API.
+Each client trains a local sub-forest on their data, and these are aggregated on the server.
+
+Usage:
+    python job.py --n_clients 5 --local_subsample 0.5 --data_split_path /tmp/nvflare/random_forest/HIGGS/data_splits/5_uniform
+"""
+
+import argparse
+
+from nvflare.app_opt.xgboost.higgs_data_loader import HIGGSDataLoader
```

Comment:
do you think this file should be here?
or inside the example folder?
this is subjective I guess I could not decide